docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+* A running Teleport Enterprise cluster v15.0.0 or later.
+* Docker version v(=docker.version=) or later.
+* A PostgresSQL database server v14 or later.
```

Comment:
I validated this by running integration tests with containerized Postgres v14. https://github.com/gravitational/access-graph/pull/429 introduces a Postgres 14+ feature (recursive CTEs), so in a very near future we will not be supporting 13 and earlier.